### PR TITLE
Add support to get all values from drop down

### PR DESCRIPTION
**Pull request by saikrishna321**
New API to return an array with all the options from a dropdown

```
await dropDown('Vehicle:').options()
```

### CODE REVIEW

**Comment by NivedhaSenthil** [comment]
I feel this can be named `options` instead of `values` as value is generally linked to selected option of the dropdown.

**Comment by zabil** [comment]
> I feel this can be named `options` instead of `values` as value is generally linked to selected option of the dropdown.

Ah yes, I agree. `options` is the right name for this API and how users refer to it.

**Comment by zabil** [comment]
>  I feel this can be named `options` instead of `values` as value is generally linked to selected option of the dropdown.

@saikrishna321 ran into an issue with the name `options` this is because 

https://github.com/getgauge/taiko/blob/c082b8c4060cbab4484d3741fae9c55da95d0725/lib/elementWrapper/elementWrapper.js#L27-L30

sets the `options` variable so calling `dropDown({ id: 'select' }).options()` will refer to `this.options` variable instead. We should either change `const { selector, options } = prepareParameters(attrValuePairs, _options, ...args); ` to something else or call the method `dropDown({ id: 'select' }).valueOptions()`

File: lib/taiko.js [hunk 1]
```diff
@@ -2488,6 +2488,8 @@ function textBox(labelOrAttrValuePairs, _options, ...args) {
  * @example
  * await dropDown('Vehicle:').value()
  * @example
+ * await dropDown('Vehicle:').values() - Returns all the values from the drop down
```
Comment (zabil) [inline lib/taiko.js:2491]: shouldn't this be 
```suggestion
 * await dropDown('Vehicle:').options() - Returns all available options from the drop down
```

File: lib/elements/dropDown.js [hunk 2]
```diff
@@ -96,6 +96,21 @@ class DropDown extends Element {
     });
     return result.value;
   }
+
+  async options() {
+    function getValues() {
```
Comment (zabil) [inline lib/elements/dropDown.js:101]:
```suggestion
    function getOptions() {
```

File: lib/elements/dropDown.js [hunk 3]
```diff
@@ -96,6 +96,21 @@ class DropDown extends Element {
...
+      let dropDownValues = [];
+      for (let index = 0; index < this.options.length; index++) {
+        dropDownValues.push(this.options[index].value);
+      }
+      return dropDownValues;
+    }
+    const { result } = await this.runtimeHandler.runtimeCallFunctionOn(getValues, null, {
```
Comment (zabil) [inline lib/elements/dropDown.js:108]:
```suggestion
    const { result } = await this.runtimeHandler.runtimeCallFunctionOn(getOptions, null, {
```

File: test/unit-tests/dropDown.test.js [hunk 4]
```diff
@@ -132,6 +132,15 @@ describe(test_name, () => {
       expect(await dropDown('Cars').value()).to.equal('mercedes');
     });
 
+    it('test values()', async () => {
```
Comment (zabil) [inline test/unit-tests/dropDown.test.js:135]:
```suggestion
    it('test options()', async () => {
```

**Review by zabil — CHANGES_REQUESTED** [review]

**Review by zabil — APPROVED** [review]

**Pull request merged**